content/docs/howto/dotnet-core.md

```diff
@@ -315,6 +315,94 @@ access the SBOM supplied by the buildpacks.
 
 SBOMs will be generated for all supported .NET Core applications.
 
+## Enable Remote Debugging
+
+Remote debugging can provide insight into complex program logic and
+interactions in remote environments. This practice is supported for .NET Core
+applications via the Visual Studio Debugger (`vsdbg`), which may be included in
+your application image via the BP_DEBUG_ENABLED environment variable. The
+debugger can attach to a running dotnet process and be bound to a client-side
+debugger via STDIN across a connection invoked via `docker exec` or `kubectl exec`.
+
+### Using `BP_DEBUG_ENABLED`
+
+To enable remote debugging, set the `$BP_DEBUG_ENABLED` environment
+variable at build time, either by passing a flag to the
+[platform][definition/platform] or by
+adding it to your `project.toml`. See the Cloud Native Buildpacks
+[documentation][project-file] to learn more about `project.toml` files.
+
+#### With a `pack build` flag
+<!-- spellchecker-disable -->
+{{< code/copyable >}}
+pack build myapp --env BP_DEBUG_ENABLED=true
+{{< /code/copyable >}}
+<!-- spellchecker-enable -->
+
+#### In a `project.toml` file
+<!-- spellchecker-disable -->
+{{< code/copyable >}}
+[[ build.env ]]
+  name = 'BP_DEBUG_ENABLED'
+  value = 'true'
+{{< /code/copyable >}}
+<!-- spellchecker-enable -->
+
+### Setting up Visual Studio Code for Remote Debugging
+
+Visual Studio Code can be configured to attach a remote debugging session into
```

Comment:
Are there any relevant Visual Studio Code related docs for configuration? Might be helpful to just add a link if you are aware of any.